scripts.js

```diff
@@ -18,10 +18,12 @@ import fs from 'fs';
 import yargs from 'yargs';
 import {hideBin} from 'yargs/helpers';
 
-const exec = (cmd, env) => {
+export const exec = (cmd, env, pipe = true) => {
   env = {...process.env, ...env};
-  console.log(`> ${cmd}`);
-  return childProcess.execSync(cmd, {env, stdio: 'inherit'});
+  if (!pipe) {
+    console.log(`> ${cmd}`);
```

Comment:
Is this leftover debugging? It looks a lot like the format that `npm run bla` prints, but why print anything?

Reply:
This function is used by `npm run prepare` and `npm test` as well, which is why there's this console log.